.eslintignore

```diff
@@ -1 +1,2 @@
 /dist
+**/*.js
```

Comment:
Is there a reason for ignoring javascript files? 🤔

Reply:
This `https://github.com/nimblehq/infrastructure-templates/blob/develop/test/helpers/init.js` JS file is not using the new code, but we will have another task to improve the testing part, so exclude them for now